encfs/RawFileIO.cpp

```diff
@@ -206,8 +209,6 @@ ssize_t RawFileIO::read(const IORequest &req) const {
 
   if (readSize < 0) {
     int eno = errno;
-    errno = 0;  // just to be sure error seen in integration tests really comes
-                // from the function rc.
```

Comment:
I would have been pleased to keep this, as explained in the comment just to be sure...
Thank you 👍